src/isaw.theme/isaw/theme/__init__.py

```diff
@@ -2,10 +2,12 @@
 from .patches import img_tag_no_title
 from .patches import ofs_img_tag_no_title
 from .patches import named_file_image_tag
+from .patches import image_csrf_fix
```

Comment:
Why doesn't every Plone site need this patch since image scaling is generally a DB write on GET/read? Maybe I'm not clear on what it does.

Reply:
This is a very old Plone site. All Plone 4.3 sites do need this patch if they generate image scales using the `@@images` from python.